client/landing/stepper/declarative-flow/design-first.ts

```diff
@@ -230,7 +238,8 @@ const designFirst: Flow = {
 		} else if ( userAlreadyHasSites && isSiteCreationStep ) {
 			// Redirect users with existing sites out of the flow as we create a new site as the first step in this flow.
 			// This prevents a bunch of sites being created accidentally.
-			redirect( `/post?${ DESIGN_FIRST_FLOW }=true` );
+			redirect( `/themes/${ primaryDomain }` );
```

Comment:
What are your thoughts on disregarding `primaryDomain` and redirecting to `/themes` instead? That would send the user to the site selection screen, then after they select the site they want, they are redirected to `/themes/{slug}`.

Reply:
My goal has to avoid that redirect in the second 1 of the video. But thinking again, it's probably better to have that so users can easily select which domain they want to choose a theme for.

https://github.com/Automattic/wp-calypso/assets/1044309/1b010c6c-0760-4570-9544-64490b887d33

Reply:
[Changed it here](https://github.com/Automattic/wp-calypso/pull/77701/commits/20c43525b0d950a3d1cd40e0e4106d25807d2e4e).

Reply:
Interesting thing about that redirect screen flash is I see it on localhost, but not in production. 🤔 I'm keen to merge this and see if it's still an issue in production.